.env.development

```diff
@@ -7,3 +6,6 @@ OKP4_TELEGRAM_URL='https://t.me/okp4network'
 OKP4_TWITTER_URL='https://twitter.com/OKP4_Protocol'
 OKP4_WEBSITE_URL='https://okp4.network/'
 DATA_PATH='./data'
+API_URI='http://localhost:3000/api'
+NEXT_PUBLIC_SERVER_URI='http://localhost:3000'
+NEXT_PUBLIC_DEFAULT_DATASPACE_ID='ef347285-e52a-430d-9679-dcb76b962ce7'
```

Comment:
I don't like this, because the values are inlined at build time, so even if we change it afterwards, it will not work. Also we lose control over the deployment.

Reply:
I know that injecting config parameters is tricky with Next.js, but it's important not to lose the ability to re-configure from a deployment perspective (especially because we have more different environments).

FYI @amimart @fredericvilcot